src/blocks/buttons/styling.js

```diff
@@ -177,9 +180,13 @@ function styling( attributes, clientId, name, deviceType ) {
 		selectors[ '.uagb-editor-preview-mode-desktop .block-editor-inner-blocks' ] = {
 			'text-align': align,
 		};
+		selectors['.uagb-editor-preview-mode-desktop .block-editor-block-list__layout'] = {
```

Comment:
For consistency, let's keep spacing in all the new lines added to this file, like so:

```suggestion
		selectors[ '.uagb-editor-preview-mode-desktop .block-editor-block-list__layout' ] = {
```

Reply:
Done